packages/twenty-front/src/modules/object-record/record-board/record-board-card/components/RecordBoardCellFieldInput.tsx

```diff
@@ -0,0 +1,73 @@
+import { useRecordBoardColumnContextOrThrow } from '@/object-record/record-board/record-board-column/contexts/RecordBoardColumnContext';
+import { FieldInput } from '@/object-record/record-field/components/FieldInput';
+import { FieldContext } from '@/object-record/record-field/contexts/FieldContext';
+import { useIsFieldValueReadOnly } from '@/object-record/record-field/hooks/useIsFieldValueReadOnly';
+import { FieldInputEvent } from '@/object-record/record-field/types/FieldInputEvent';
+import { getDropdownFocusIdForRecordField } from '@/object-record/utils/getDropdownFocusIdForRecordField';
+import { getRecordFieldInputId } from '@/object-record/utils/getRecordFieldInputId';
+import { activeDropdownFocusIdState } from '@/ui/layout/dropdown/states/activeDropdownFocusIdState';
+import { useContext } from 'react';
+import { useRecoilCallback } from 'recoil';
+
+export const RecordBoardCellFieldInput = () => {
```

Comment:
I would name this component differently. I feel it's RecordBoardPendingTextIdentifierFieldInput